### PR TITLE
Created goal post for next Tuesday

### PR DESCRIPTION
**Pull request by gunderodd**

### CODE REVIEW

File: content/resources.md [hunk 1]
```diff
@@ -15,68 +15,84 @@ weight: 20
 
 # Course
   * We are currently studying the material from {{< courselink >}}.
+  ---
```
Comment (imattman) [inline content/resources.md]: I'm pretty sure the update to this file was unintentional :-)

The preview shows the effect on the *Resources* page: https://deploy-preview-87--pythonpirates.netlify.com/resources/

Comment (gunderodd) [reply content/resources.md]: Yep, that was an accident. Is there a way to just update with my new post but not change that file? Is that something I should do by fixing my local copy and submitting a pull request again, or can we just pick what parts to update here on github?

Comment (imattman) [reply content/resources.md]: So long as this PR remains open, any changes you make to your branch on your fork will be reflected here.

Undo the change to `resources.md` on your local branch `next-week-goals` and push it to your fork.  Once that's done this PR will update to show only **1** file change.

This should revert the file content:
```
git checkout origin/master content/resources.md

git add content/resources.md

git commit -m "restored resources"
```

**Comment by imattman** [comment]
Looks much better 👍 
- only one file changed
- preview of resources looks back to normal  https://deploy-preview-87--pythonpirates.netlify.com/resources/

**Pull request merged**